homeassistant/components/flume/binary_sensor.py

```diff
@@ -0,0 +1,103 @@
+"""Alert sensors."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    DOMAIN,
+    FLUME_AUTH,
+    NOTIFICATION_BRIDGE_DISCONNECT,
+    NOTIFICATION_HIGH_FLOW,
+    NOTIFICATION_LEAK_DETECTED,
+)
+from .coordinator import FlumeNotificationDataUpdateCoordinator
+from .entity import FlumeEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class FlumeBinarySensorRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    event_rule: str
+    reverse_output: bool
+
+
+@dataclass
+class FlumeBinarySensorEntityDescription(
+    BinarySensorEntityDescription, FlumeBinarySensorRequiredKeysMixin
+):
+    """Describes a binary sensor entity."""
+
+
+FLUME_BINARY_SENSORS: tuple[FlumeBinarySensorEntityDescription, ...] = (
+    FlumeBinarySensorEntityDescription(
+        key="leak",
+        name="Leak detected",
+        event_rule=NOTIFICATION_LEAK_DETECTED,
+        icon="mdi:pipe-leak",
+        reverse_output=False,
+    ),
+    FlumeBinarySensorEntityDescription(
+        key="flow",
+        name="High flow",
+        event_rule=NOTIFICATION_HIGH_FLOW,
+        icon="mdi:waves",
+        reverse_output=False,
+    ),
+    FlumeBinarySensorEntityDescription(
+        key="bridge",
+        name="Bridge disconnected",
```

Comment:
Given that there is a `CONNECTIVITY` device class should this just be renamed to `Bridge` instead or `Bridge disconnected`

Reply:
I think the name is fine

Reply:
It looks kinda dumb though...

<img width="486" alt="image" src="https://user-images.githubusercontent.com/6491743/185646843-9b6e50ec-5239-4050-b88a-8b49935d9c43.png">

.... I think I'm changing it :)